experimental/federated_plugin/federated_plugin/example/pubspec.lock

```diff
@@ -162,21 +174,21 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.17"
+    version: "0.2.18"
   typed_data:
     dependency: transitive
     description:
       name: typed_data
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.3.0-nullsafety"
+    version: "1.3.0-nullsafety.2"
   vector_math:
     dependency: transitive
     description:
       name: vector_math
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.0-nullsafety"
+    version: "2.1.0-nullsafety.2"
 sdks:
-  dart: ">=2.9.0-18.0 <2.9.0"
+  dart: ">=2.10.0-0.0.dev <2.10.0"
```

Comment:
Hey @domesticmouse, the sample is currently targetting the master branch of flutter since we are also looking forward to adding support for the windows.

Reply:
Hey @domesticmouse, I have downgrade dependencies to the beta channel for every package. I'll update it to master for specific package as I'm adding the support for platforms which requires dev or master.

Reply:
One of the organizational decisions we made a while back was that everything in `/experimental` would target master for simplicity's sake:

https://github.com/flutter/samples/tree/master/experimental

If that needs to change, we can certainly change it, but we should talk it over first.